src/mgmt/srv_drpc.c

```diff
@@ -119,6 +119,27 @@ ds_mgmt_drpc_kill_rank(Drpc__Call *drpc_req, Drpc__Response *drpc_resp)
 	mgmt__kill_rank_req__free_unpacked(req, NULL);
 }
 
+void
+ds_mgmt_drpc_ping_rank(Drpc__Call *drpc_req, Drpc__Response *drpc_resp)
```

Comment:
I think it would be trivial to add a unit test for the success case

Reply:
will do